components/ApplicationContainer/ApplicationContainer.tsx

```diff
@@ -1,31 +1,36 @@
 import { AppShell } from "@mantine/core";
+import { useRouter } from "next/router";
 
 import BottomNavigation from "../BottomNavigation/BottomNavigation";
 import PostSheet from "../PostSheet/PostSheet";
 import FileDropOverlay from "../FileDropOverlay/FileDropOverlay";
 
 export const ApplicationContainer = ({
   children,
-}: React.PropsWithChildren<{}>) => (
-  <AppShell
-    padding={0}
-    styles={{
-      root: {
-        width: "100%",
-        maxWidth: "600px",
-        height: "100vh",
-        margin: "auto",
-      },
-      main: {
-        width: "100%",
-        height: "100vh",
-      },
-    }}
-    fixed
-    footer={<BottomNavigation />}
-  >
-    {children}
-    <PostSheet />
-    <FileDropOverlay />
-  </AppShell>
-);
+}: React.PropsWithChildren<{}>) => {
+  const router = useRouter();
+
+  return (
+    <AppShell
+      padding={0}
+      styles={{
+        root: {
+          width: "100%",
+          maxWidth: router.pathname === "/" ? "100%" : 600,
```

Comment:
I needed the width to be 100% on the main page so that the scrollbar stays on the right edge of the screen.

Reply:
You can replace "/" with Route.Home